CucumberDemo/src/test/resources/features/Test_android_web.feature

```diff
@@ -0,0 +1,21 @@
+@androidWeb @web @demoCucumber
```

Comment:
1. Duplicated scenarios testing web on android and iOS. Should merge Test_android_web.feature, Test_ios_app.feature into one.
2. The same as app testing, need one scenario for both android and iOS which more realistic.

Reply:
1. That's my concern when I start to write script web test. But there are some reasons that need to separate them:
- This is just a sample test. If we include android and ios web into a file, the script will be more complex (call driver, fill suitable desire capability, test runner)
- User has to replace, config many variables and files to run a test.
2. The same app testing but there are many differences in the element, GUI on android and ios devices, you can see more detail in scenarios.